code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -547,8 +547,10 @@
 	var/distance = get_dist(src, H)
 
 	if (distance > 2)
+		to_chat(src, "<span class='xenowarning'>Your target is too far away!</span>")
```

Comment:
this can result in spam when trying to use it; is this info necesary?

Reply:
I'd say it's definitely helpful. They'd only be spamming themselves.

Reply:
Yes, I'd remove that as well.